src/psyclone/psyGen.py

```diff
@@ -1199,26 +1213,49 @@ def parent(self, my_parent):
 
     @property
     def position(self):
+        '''
+        Find a Node's position relative to its parent Node (starting
+        with 0 if it does not have a parent).
+        :returns: relative position of a Node to its parent
+        :rtype: int
+        '''
         if self.parent is None:
-            return 0
+            return self.START_POSITION
```

Comment:
This line is not covered.

Reply:
Added the appropriate test in `test_node_position` (`psyGen_test.py`).